src/screens/Dashboard/index.tsx

```diff
@@ -1,30 +1,62 @@
 import React from 'react';
 import {createStackNavigator, TransitionPresets} from '@react-navigation/stack';
-import Dashboard from './Dashboard';
 import TripSearch from '@atb/screens/Dashboard/TripSearch';
+import {TripDetailsRoot} from '../TripDetails';
+import {SearchTime} from '../Nearby/types';
+import {Location} from '@atb/favorites/types';
+import {DetailsRouteParams} from '../TripDetails/Details';
+import JourneyDatePicker, {
+  DateTimePickerParams,
+} from '../Assistant/journey-date-picker';
+import DashboardRoot from './Dashboard';
 
 export type DashboardParams = {
-  Dashboard: undefined;
-  TripSearch: undefined;
+  DashboardRoot: {
+    fromLocation: Location | undefined;
+    toLocation: Location | undefined;
+    searchTime: SearchTime | undefined;
+  };
```

Comment:
This type could be a named concept. But we can handle it when refactoring next week

Reply:
:+1: